src/constants/constants.ts

```diff
@@ -52,3 +55,9 @@ export enum SortingLabels {
   TO_LOW = 'Price high to low',
   BY_ALPHABET = 'By alphabet',
 }
+
+export enum ACTIONS {
+  addLineItem = 'addLineItem',
+  changeLineItemQuantity = 'changeLineItemQuantity',
+  removeLineItem = 'removeLineItem',
+}
```

Comment:
Perhaps it would be better to use UPPER_SNAKE_CASE for vars name.
```suggestion
export enum ACTIONS {
  ADD_LINE_ITEM = 'addLineItem',
  CHANGE_LINE_ITEM_QUANTITY = 'changeLineItemQuantity',
  REMOVE_LINE_ITEM = 'removeLineItem',
}
```